pkgs/servers/sozu/default.nix

```diff
@@ -13,7 +13,7 @@ rustPlatform.buildRustPackage rec {
 
   cargoSha256 = "5WOigCiQZQ5DaTd15vV8pUh8Xl3UIe9yLG1ptUtY+iA=";
 
-  buildInputs = stdenv.lib.optionals stdenv.isDarwin
+  buildInputs = stdenv.lib.optional stdenv.isDarwin
     [ darwin.apple_sdk.frameworks.Security ];
```

Comment:
```suggestion
    darwin.apple_sdk.frameworks.Security;
```

Reply:
I thought buildInputs had to be an array. Didn't know it could work without.
Thanks for the tip.

Reply:
Yes, `optional` returns an array as well, for more details look at https://github.com/NixOS/nixpkgs/blob/7db09ea70a39f153536bc651984765e626a2ca1b/lib/lists.nix#L242-LL270